test/regression/regression_test.cpp

```diff
@@ -23,7 +23,7 @@
 #include "framework/integration_framework/integration_test_framework.hpp"
 #include "framework/specified_visitor.hpp"
 
-constexpr auto kUser = "user@test";
+constexpr auto kAdmin = "user@test";
```

Comment:
`kAdmin` with credentials of the user? Should be changed, I think

Reply:
oh my.. It was unintentionally refactored